templates/index.html

```diff
@@ -28,13 +28,30 @@
         <div class="header-inner">
           <div class="row">
             <div class="header-left col-md-2">
-              <a href="#" @click="submittex" class="primary-action">TeX</a>
+              <a href="#" @click="clearContent" class="primary-action">
+                <div class="buttontext">
+                  Clear
+                </div>
+              </a>
             </div>
             <div class="col-md-8">
-              <h2>Motioner</h2>
             </div>
             <div class="header-right col-md-2">
-              <a href="#" @click="submitpdf" class="primary-action">PDF</a>
+              <a href="#" @click="submitpdf" class="primary-action">
+                <div class="buttontext">
+                  PDF
+                </div>
```

Comment:
Does creating PDF work? I had problems with it last week.

Reply:
Yes, it was fixed when deploying #14

Reply:
There are some unintuitive behaviour/bugs where some inputs results in PDF creation failing though, which I intend to look at at some later point